src/main/assets/js/court-types.ts

```diff
@@ -0,0 +1,83 @@
+import $ from 'jquery';
+import tinymce from 'tinymce';
+const { initAll } = require('govuk-frontend');
+
+export class CourtTypesController {
+  private formId = '#courtTypesForm';
+  private tabId = '#courtTypesTab';
+  private courtTypesContentId = '#courtTypesContent';
+
+
+  constructor() {
+    this.initialize();
+
+  }
+
+  private initialize(): void {
+    $(() => {
+      if ($(this.tabId).length > 0) {
+        this.getCourtTypes();
+        this.setUpSubmitEventHandler();
+      }
+    });
+  }
+
+
+  private async updateContent(content: any): Promise<void> {
+    $(this.courtTypesContentId).html(content);
+
+    tinymce.remove();
+    await tinymce.init({
+      selector: '.rich-editor',
+      plugins: 'autolink link paste',
+      menubar: '',
+      toolbar: 'link bold italic underline',
+      'paste_as_text': true,
+      statusbar: false,
+    });
```

Comment:
Is this text editor needed for court types?

Reply:
Removed not needed